.github/workflows/create_example_nbs.yml

```
@@ -0,0 +1,39 @@
+name: "Convert Examples to Notebooks"
```

```suggestion
name: "Check Example Notebooks"
```

Could you also update the `create_example_nbs.yml` filename accordingly?